examples/in-source-test/vite.config.ts

```diff
@@ -0,0 +1,11 @@
+/// <reference types="vitest" />
+import { defineConfig } from 'vite'
```

Comment:
Let's name it `vitest.config.ts` and use `vitest/config` since it doesn't anything from vite

Reply:
Replaced vite.config.ts with vitest.config.ts and modified import.
[74e13ac](https://github.com/vitest-dev/vitest/pull/6259/commits/74e13ac3e018e25c6f7a822b977e2bee4c3b66a3)